.github/workflows/release-prep.yml

```diff
@@ -186,24 +187,48 @@ jobs:
         with:
           ref: ${{needs.create-new-branch.outputs.BRANCH_NAME}}
```

Comment:
```suggestion
          ref: ${{ needs.create-new-branch.outputs.BRANCH_NAME }}
```

Reply:
> Looks good. Can we add the PYTHON_VERSION env variable to this workflow like you have in the others?

Sure, I will add the `PYTHON_VERSION` env variable.

Reply:
Resolved in 8efd280ed1ded88cb1a886a0d6a089dc84c0155e